pokemongo_bot/api_wrapper.py

```diff
@@ -1,7 +1,7 @@
 # api_wrapper.py
 
 from pgoapi import PGoApi
-from pgoapi.exceptions import NotLoggedInException
+from pgoapi.exceptions import NotLoggedInException, ServerBusyOrOfflineException
```

Comment:
All good. It throws an exception now and it actually increments. Earlier it just repeatedly yielded attempt 1 of 5. I'm going to get the retry count passed in from a config and resubmit.